plugins/k8saudit/pkg/k8saudit/source.go

```diff
@@ -50,11 +51,44 @@ func (k *Plugin) Open(params string) (source.Instance, error) {
 	case "https":
 		return k.OpenWebServer(u.Host, u.Path, true)
 	case "": // by default, fallback to opening a filepath
-		file, err := os.Open(params)
+		trimmed := strings.TrimSpace(params)
+
+		fileInfo, err := os.Stat(trimmed)
+		if err != nil {
+			return nil, err
+		}
+		if !fileInfo.IsDir() {
+			file, err += os.Open(trimmed)
+			if err != nil {
+				return nil, err
+			}
+			return k.OpenReader(file)
+		}
+
+		files, err := ioutil.ReadDir(trimmed)
 		if err != nil {
 			return nil, err
 		}
-		return k.OpenReader(file)
+
+		sort.Slice(files, func(i, j int) bool {
+			return files[i].ModTime().Before(files[j].ModTime())
+		})
+
+		// open all files as reader
+		results := []io.Reader{}
+		for _, f := range files {
+			if !f.IsDir() {
+				auditFile, err := os.Open(f.Name())
```

Comment:
```suggestion
				auditFile, err := os.Open(trimmed + "/" + f.Name())
```
We need to add the base file path, otherwise this will always fail (just tested it locally).